pkg/uptimerobot.go

```diff
@@ -126,12 +153,14 @@ type Monitor struct {
 	Port          interface{} `json:"port"`
 	KeywordValue  string      `json:"keyword_value"`
 	AlertContacts []string    `json:"alert_contacts"`
+	Status        Status      `json:"status"`
```

Comment:
Do we need to add `omitempty` here? When we send a monitor create request, we probably don't want to send a blank status field too, though I dare say it doesn't matter. (I didn't learn about `omitempty` until after I'd done the first iteration of this library.)